source/installation/install_from_source.rst

```diff
@@ -232,36 +236,3 @@ don't go smooth. In some rare cases, enabling debug logging (-d option)
 in rsyslogd can be helpful. If all fails, go to
 `www.rsyslog.com <http://www.rsyslog.com>`_ and check the forum or
 mailing list for help with your issue.
-
-Housekeeping stuff
```

Comment:
This section (lines 235-267) felt like it was in need of pruning. The contact information is already provided in many places, the footer now includes links to multiple "go here for help" resources and the revision history doesn't seem particularly relevant any longer (the Git history is available to anyone who is really interested in the history of a document).

Reply:
ACK - I need to add some info on why some of these constructs are there. It's actually contributed pages. These is mine, so it is not a problem. Will document the so far non-written policy soon. Good this came up!